rules_gathering/BUILD

```diff
@@ -18,6 +18,8 @@ package(
     default_visibility = ["//visibility:public"],
 )
 
+load(":toolchains.bzl", "sbom_tools_toolchain")
```

Comment:
Nit: I think there is a buildifier lint rule that mandate `load` statements must be on top of BUILD files?